src/components/create-study-form.js

```diff
@@ -268,12 +268,14 @@ export const CreateStudyForm = () => {
             return;
         }
 
-        let isPrivateStudy = false;
-        if (studyPrivacy === 'private') {
-            isPrivateStudy = true;
-        }
+        let isPrivateStudy = studyPrivacy === 'private';
 
         setOpen(false);
+        props.addCreationRequest({
+            studyName: studyName,
+            userId: userId,
+            isPrivate: isPrivateStudy,
+        });
```

Comment:
Parameter "date" is not set when calling addCreationRequest, is it normal ? When the temporary study card is displayed, an "Invalid date" label is then displayed.